daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/LanguageMajorVersion.scala

```diff
@@ -44,7 +44,13 @@ object LanguageMajorVersion {
         minorAscending = NonEmptyList("6", "7", "8", "11", "12", "13", "14", "15", "dev"),
       )
 
-  val All: List[LanguageMajorVersion] = List(V1)
+  case object V2
+      extends LanguageMajorVersion(
+        pretty = "2",
+        minorAscending = NonEmptyList("dev"),
```

Comment:
I think we should create v2.0 now too

Reply:
As discussed we'll do that in a second step, when we decide to break backwards compatibility between v2 and v1.